src/FormattingToolbar/toolbarTooltip.js

```diff
@@ -5,28 +5,60 @@
 /**
  * Methods for conversion to strings and further manipulation.
  */
+
 const capitalizeFirst = word => word.toString().charAt(0).toUpperCase();
 
 const sliceWord = word => word.toString().slice(1);
 
 const firstTwoLetters = word => word.toString().slice(0, 2);
 
+
+/**
+ * Function to determine OS and MOD command of user
+ */
+
+function OS () {
+  const {platform} = window.navigator;
+
+  const macosPlatforms = {
+    os: ["Macintosh", "MacIntel", "MacPPC", "Mac68K"],
+    MOD: "⌘"
+  };
+
+  const windowsPlatforms = {
+    os : ["Win32", "Win64", "Windows", "WinCE"],
+    MOD: "Ctrl"
+  };
+
+  if (macosPlatforms.os.indexOf(platform) !== -1) {
+    return macosPlatforms.MOD;
+  }  else if (windowsPlatforms.os.indexOf(platform) !== -1) {
+    return windowsPlatforms.MOD;
+  } else if (!os && /Linux/.test(platform)) {
+    return windowsPlatforms.MOD;
+  }
+}
+
+export const MOD = OS();
```

Comment:
I'm thinking this can still go further, like:

```js
  const macosPlatforms = {
    os: {
      "Macintosh": true,
      "MacIntel": true,
      "MacPPC": true,
      "Mac68K": true
    },
    MOD: "⌘"
  };
```

You could then not need `.indexOf` which needs to iterate through the array, but rather look up by key value.

Reply:
```
function OS () {
  const {platform} = window.navigator;

  const macosPlatforms = {
    os: {
      "Macintosh": true, 
      "MacIntel": true, 
      "MacPPC": true, 
      "Mac68K": true
    },
    MOD: "⌘"
  };

  const windowsPlatforms = {
    os : {
      "Win32": true, 
      "Win64": true, 
      "Windows": true, 
      "WinCE": true
    },
    MOD: "Ctrl"
  };

  if (macosPlatforms.os[platform] !== undefined) {
    return macosPlatforms.MOD;
  }  else
    return windowsPlatforms.MOD;
}

```
@irmerk would this be a correct implementation of the same? I works fine though.

Reply:
Yes I believe so: 

```js
 if (macosPlatforms.os[platform]) return macosPlatforms.MOD;
 if (windowsPlatforms.os[platform]) return windowsPlatforms.MOD;
 return windowsPlatforms.MOD;
```

Reply:
Yupp, thanks. I'll push the changes